flow/designs/asap7/sha3/metadata-base-ok.json

```diff
@@ -25,97 +25,97 @@
   "cts__timing__drv__max_cap__pre_repair": 0.0,
-  "cts__timing__setup__ws__pre_repair": -335.67,
-  "cts__timing__setup__tns__post_repair": -9721.37,
-  "cts__timing__setup__ws__post_repair": -335.67,
-  "cts__timing__drv__max_slew__post_repair": 0.0,
+  "cts__timing__setup__ws__pre_repair": -345.03,
+  "cts__timing__setup__tns__post_repair": -10021.58,
+  "cts__timing__setup__ws__post_repair": -345.03,
+  "cts__timing__drv__max_slew__post_repair": 41.0,
   "cts__timing__drv__max_fanout___post_repair": 0.0,
   "cts__timing__drv__max_cap__post_repair": 0.0,
-  "cts__timing__setup__tns": -11.06,
-  "cts__timing__setup__ws": -7.65,
-  "cts__timing__drv__max_slew": 9.0,
+  "cts__timing__setup__tns": 0.0,
+  "cts__timing__setup__ws": 7.77,
+  "cts__timing__drv__max_slew": 0.0,
   "cts__timing__drv__max_fanout": 0.0,
-  "cts__design__instance__count__hold_buffer": 114.0,
-  "globalroute__clock__latency__min": 117.56,
-  "globalroute__clock__latency__max": 143.1,
-  "globalroute__clock__skew__worst": 25.54,
-  "globalroute__timing__setup__tns": -86.81,
-  "globalroute__timing__setup__ws": -14.89,
-  "globalroute__timing__drv__max_slew": 41.0,
+  "cts__design__instance__count__hold_buffer": 132.0,
+  "globalroute__clock__latency__min": 121.51,
+  "globalroute__clock__latency__max": 167.49,
+  "globalroute__clock__skew__worst": 45.98,
+  "globalroute__timing__setup__tns": -3.97,
+  "globalroute__timing__setup__ws": -3.97,
+  "globalroute__timing__drv__max_slew": 0.0,
   "globalroute__timing__drv__max_fanout": 0.0,
   "globalroute__timing__drv__max_cap": 0.0,
-  "globalroute__timing__clock__slack": -14.893,
-  "globalroute__timing__clock__period": 660.149,
-  "globalroute__runtime__total": "0:13.45",
-  "globalroute__cpu__total": 12.64,
-  "globalroute__mem__peak": 429812.0,
-  "detailedroute__route__wirelength": 69260.0,
-  "detailedroute__route__vias": 157415.0,
+  "globalroute__timing__clock__slack": -3.97,
+  "globalroute__timing__clock__period": 649.771,
+  "globalroute__runtime__total": "0:11.07",
+  "globalroute__cpu__total": 10.72,
+  "globalroute__mem__peak": 465204.0,
+  "detailedroute__route__wirelength": 67995.0,
+  "detailedroute__route__vias": 156520.0,
   "detailedroute__route__drc_errors": 0,
-  "detailedroute__runtime__total": "5:16.36",
-  "detailedroute__cpu__total": 2664.52,
-  "detailedroute__mem__peak": 7569176.0,
-  "finish__clock__latency__min": 116.89,
-  "finish__clock__latency__max": 141.61,
-  "finish__clock__skew__worst": 24.47,
-  "finish__timing__setup__tns": -4903.99,
-  "finish__timing__setup__ws": -180.62,
-  "finish__timing__drv__max_slew": 126.0,
+  "detailedroute__runtime__total": "8:02.33",
+  "detailedroute__cpu__total": 2548.04,
+  "detailedroute__mem__peak": 5263468.0,
+  "finish__clock__latency__min": 122.54,
+  "finish__clock__latency__max": 167.76,
+  "finish__clock__skew__worst": 44.82,
+  "finish__timing__setup__tns": -1650.9,
+  "finish__timing__setup__ws": -37.08,
+  "finish__timing__drv__max_slew": 663.0,
```

Comment:
This is a high increase on `finish__timing__drv__max_slew`.

Reply:
Yes  max_slew has been very noisy.  As this is a flow change. I would let this into CI and look at max_slew and hold buffer count to bring those under control